core/src/main/kotlin/org/opensearch/alerting/core/settings/LegacyOpenDistroScheduledJobSettings.kt

```diff
@@ -0,0 +1,48 @@
+package org.opensearch.alerting.core.settings
+
+import org.opensearch.common.settings.Setting
+import org.opensearch.common.unit.TimeValue
+
+/**
+ * Legacy Opendistro settings used for [ScheduledJob]'s. These include back off settings, retry counts, timeouts etc...
+ */
+
+class LegacyOpenDistroScheduledJobSettings {
+
+    companion object {
+        val SWEEPER_ENABLED = Setting.boolSetting(
+            "opendistro.scheduled_jobs.enabled",
+            true,
+            Setting.Property.NodeScope, Setting.Property.Dynamic, Setting.Property.Deprecated
+        )
+        val REQUEST_TIMEOUT = Setting.positiveTimeSetting(
```

Comment:
Add a line break above ;)

Reply:
Adding klint formatting to the whole project. It will take care of all these changes for future. ;)

Reply:
@aditjind we got ktlint enabled as part of gradle build. Is that missing now?

Reply:
Updating the version and applied the formatting. (https://github.com/opensearch-project/alerting/pull/20)